web-ui/src/api/birthdayanniversary.js

```diff
@@ -4,23 +4,33 @@ const anniversaryReportUrl = "/services/reports/anniversaries";
 const birthdayReportUrl = "/services/reports/birthdays";
 const celebrationsToday = "/services/today";
 
-export const getAnniversary = async (month, cookie) => {
-  return resolve({
-    url: `${anniversaryReportUrl}?month=${month}`,
-    headers: { "X-CSRF-Header": cookie, "Accept": "application/json" },
-  });
+export const getAnniversaries = async (months, cookie) => {
+  const results = [];
+  for (let month of months) {
+    const res = await resolve({
+      url: `${anniversaryReportUrl}?month=${month}`,
+      headers: { "X-CSRF-Header": cookie, Accept: "application/json" },
+    });
+    results.push(...res.payload.data);
+  }
+  return results;
 };
 
 export const getTodaysCelebrations = async (cookie) => {
   return resolve({
     url: celebrationsToday,
-    headers: { "X-CSRF-Header": cookie, "Accept": "application/json" },
+    headers: { "X-CSRF-Header": cookie, Accept: "application/json" },
   });
-}; 
+};
 
-export const getBirthday = async (month, cookie) => {
-  return resolve({
-    url: `${birthdayReportUrl}?month=${month}`,
-    headers: { "X-CSRF-Header": cookie, "Accept": "application/json" },
-  });
-};
\ No newline at end of file
+export const getBirthdays = async (months, cookie) => {
+  const results = [];
```

Comment:
This now iterates over all the selected months and accumulates the results from each month.